.eslintrc

```diff
@@ -22,5 +22,6 @@
       "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
       "react/forbid-prop-types": "off",
       "react/no-array-index-key": "off",
+      "linebreak-style": "off",
```

Comment:
Got this to work. There was[ a little git magic ](https://help.github.com/articles/dealing-with-line-endings/)to make my local version update.